### ISSUE TITLE
Custom Dataset으로도 실험할 수 있나요?

### ISSUE THREAD

**Issue by su-hyeonjeong** [kkang831/BDInvert_Release#5]
안녕하세요! 먼저, 연구 코드를 공개해주셔서 감사합니다.

제가 제 Custom Dataset (얼굴, 타워 등 X) 으로 실험을 해보려고하는데,

제 데이터셋으로도 같은 실험이 가능 한건가요 ??

(제 데이터셋으로 학습시킨 pre-trained stylegan2 model은 있습니다.)

감사합니다 :)

**Comment by kkang831** [comment]
네 가능합니다. 본 연구는 미리 학습된 모델과 데이터가 있다면 같은 방법으로 적용할 수 있기에 모델 load만 적절히하시면 basecode encoder의 training 및 이후 optimization이 가능합니다.

**Comment by su-hyeonjeong** [comment]
앗 그렇군요 감사합니다!!!! ㅎㅎ

**Comment by su-hyeonjeong** [comment]
안녕하세요!! 혹시 모델 load를 적절히 한다는게 어떤건지 알수있을까요?ㅠㅠ 가지고 있던 stylegan1,2 둘다 해당 코드에서 불러오는 네트워크와 네이밍룰이 너무 안맞아서 안돌아가다가 네이밍룰을 억지로 맞춰주니 돌아는 가는데 fake 이미지를 제대로 생성하지 못하고 노이즈만 나옵니다 ㅠㅠ.......

**Comment by su-hyeonjeong** [comment]
@kkang831 
repo 해두신 stylegan1 github에 공개된 소스로 모델을 (pkl 파일) 생성한 후 interfacegan 에 있는 convert 함수를 이용해 pth 파일로 바꾸어 사용했습니다!!

**Comment by kkang831** [comment]
저는 Genforce lib (https://github.com/genforce/genforce)에 pretrained model을 사용하였기에 from scratch로 StyleGAN을 training한 적이 없어서 정확한 답변을 드리기는 힘들 것 같지만 Genforce library에 model converter가 제공되니 이를 이용하여 converting하는 것이 적절할 것 같습니다. (https://github.com/genforce/genforce/blob/master/converters/README.md)
